src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
Remove the `else` after an `if` block that contains a `return` to simplify the flow of the function. The `else if` is not needed because if the condition in the `if` statement is true, the function will return and not proceed to the next block.